certificate-policy.md

```diff
@@ -1786,122 +1786,134 @@ During the period in which a Subordinate CA issues Certificates, the Root CA sha
 ## 9.1 Fees
 
 ### 9.1.1 Certificate issuance or renewal fees
+No stipulation.
 
 ### 9.1.2 Certificate access fees
+Section 2 of this policy requires that Repositories including CA Certificates are publicly available.  CAs operating under this policy shall not charge additional fees for access to CA Certificates.
 
 ### 9.1.3 Revocation or status information access fees
+Section 2 of this policy requires that Repositories including certificate status services (CRLs and OCSP) are publicly available.  CAs operating under this policy shall not charge additional fees for access to CRLs and OCSP services.
+
+CAs shall not charge Subscribers a fee to revoke a certificate. 
 
 ### 9.1.4 Fees for other services
+No stipulation.
 
 ### 9.1.5 Refund policy
+No stipulation.
 
 ## 9.2 Financial responsibility
+This CP contains no limits on the use of certificates issued by CAs under this policy.  Rather, entities, acting as relying parties, shall determine what financial limits, if any, they wish to impose for certificates used to consummate a transaction.
 
 ### 9.2.1 Insurance coverage
+No stipulation.
 
 ### 9.2.2 Other assets
+No stipulation.
 
 ### 9.2.3 Insurance or warranty coverage for end-entities
+No stipulation.
 
 ## 9.3 Confidentiality of business information
+CA information not requiring protection shall be made publicly available. 
 
 ### 9.3.1 Scope of confidential information
+No stipulation.
 
 ### 9.3.2 Information not within the scope of confidential information
+No stipulation.
 
 ### 9.3.3 Responsibility to protect confidential information
+A CA shall not disclose non-certificate information to any third party unless authorized by this policy, required by U.S. law, U.S. government rule or regulation, or order of a U.S. court of competent jurisdiction.  The contents of the archives maintained by CAs operating under this policy shall not be released except as required by this policy, required by U.S. law, U.S. government rule or regulation, or order of a U.S court of competent jurisdiction.  The FPKI Policy Authority must authenticate any request for release of information.
 
 ## 9.4 Privacy of personal information
 
 ### 9.4.1 Privacy plan
+CAs shall conduct a Privacy Impact Assessment, and implement and maintain any required Privacy Plans in accordance with the requirements of the Privacy Act of 1974, as amended. 
```

Comment:
Updating to be the Privacy Threshold Assessment - which may determine that a PIA and Privacy Plans are warranted, or not.

Reply:
[great feedback from chief privacy officer]